theme_kea/views/snippets/s_cover.xml

```diff
@@ -3,23 +3,32 @@
     <template id="s_cover" inherit_id="website.s_cover">
         <!-- Section -->
         <xpath expr="//section" position="attributes">
-            <attribute name="class" add="o_full_screen_height" separator=" "/>
-            <attribute name="data-oe-shape-data">{"shape":"web_editor/Origins/07_002","flip":[]}</attribute>
+            <attribute name="class" add="s_parallax_no_overflow_hidden oe_img_bg o_full_screen_height" remove="parallax s_parallax_is_fixed bg-black-50 remove" separator=" "/>
+            <attribute name="data-scroll-background-ratio">0</attribute>
+            <attribute name="data-oe-shape-data">{"shape":"web_editor/Floats/02","flip":[]}</attribute>
+            <attribute name="style">background-image: url('/web/image/website.s_cover_default_image'); background-position: 50% 0%;</attribute>
```

Comment:
`background-image` and `background-position`
Same as the snippet. This line change nothing.

Reply:
I think I need to keep this line (9) because I'm "disabling" the parallax span a bit later. Usually the `background-image` is on the `span `and not `section`, that's probably why I put it there. Without this line, the background image doesn't display at all.